### ISSUE TITLE
Module not found: Can't resolve '@mui-treasury/components/flex'

### ISSUE THREAD

**Issue by niemtin007** [siriwatknp/mui-treasury#837]
Hello Siriwatknp,

First off, I'm very appreciative of your nice project. It's saving me a lot of time.

When I tried to use your showcase component [here ](https://mui-treasury.com/components/card#Showcase) I have met an error 
> "Module not found: Can't resolve '@mui-treasury/components/flex' in ..."

I installed the newest module by using the NPM & Yarn but still met the same error. (it seems the flex module is missing!?)

![image](https://user-images.githubusercontent.com/12592676/83596659-5f89e380-a58f-11ea-8869-9c7f12d94101.png)

Please help me to fix it. 

Thanks,

**Comment by siriwatknp** [comment]
Oh, yes. I haven't publish this component yet. will do it today. sorry for that.

**Comment by siriwatknp** [comment]
just published new version, please upgrade to
```
"@mui-treasury/components": "^1.4.0",
"@mui-treasury/styles": "^1.7.0" 
```

**Comment by niemtin007** [comment]
It works well now. Thanks.

**Comment by kpennell** [comment]
@siriwatknp Really really appreciate the work you've put into this library. I ran into similar issues where I was trying to use this card I found but I had to downgrade to `"@mui-treasury/components": "^1.5.2",` to get it to work. It seems like the demos should list which version they are built with? Anyway, I don't mean to be a complainer. But you might mention in the docs which version people need for each demo.
![image](https://user-images.githubusercontent.com/1060385/106337532-39176780-6246-11eb-9441-b802473f620d.png)
